virtual-desktop/src/app/window-manager/mvd-window-manager/launchbar/launchbar-menu/launchbar-menu.component.ts

```diff
@@ -106,16 +119,27 @@ export class LaunchbarMenuComponent implements MVDHosting.LoginActionInterface{
   
   activeToggle(): void {
     this.isActive = !this.isActive;
+    // gain focus and clear on toggle when active
+    if(this.isActive) {
+      setTimeout(() => {
+        this.searchAppInputRef.nativeElement.focus();
```

Comment:
Why is this wrapped in a setTimeout?

Also, there's an HTML way of giving an element default focus when it spawns. If I remember it's the "autofocus" attribute but not sure on performance of yours vs that one

Reply:
1) settimeout so element added to DOM otherwise nativeElement was null
2) autofocus just works one time for some reason and then it dont focus any more, tried that html attribute

Reply:
Ah, I see. Thanks, good to know @NakulManchanda